src/ui/common/src/components/pages/AccountPage.tsx

```diff
@@ -103,6 +155,17 @@ client = aqueduct.Client(
   const [showNotificationUpdateSnackbar, setShowNotificationUpdateSnackbar] =
     useState(false);
 
+  useEffect(() => {
```

Comment:
This could be better served as a redux reducer. Just putting it here for now as I don't know other places in our UI where it will be used at the moment.

Reply:
Redux reducer code is coming in follow up PR, #1009